detector/fcal/LumiCal_o1_v01_geo.cpp

```diff
@@ -56,6 +44,10 @@ static Ref_t create_detector(Detector& theDetector, xml_h element, SensitiveDete
   xml_det_t xmlLumiCal = element;
   const std::string detName = xmlLumiCal.nameStr();
 
+  // only set detector like type flags if there is a sensitive element in this detector, needed to unambniguously
+  // identify the actual lumical in e.g., Pandora
+  bool have_sensitive = false;
```

Comment:
```suggestion
  bool haveSensitive = false;
```
Up to you to decide as we are a bit all over the place (already just in this file) with the spelling of variables. Usually we go with camelCase (and that also seems to be the majority here).

Reply:
True! I changed to camelCase!